packages/react-jsx-highcharts/test/components/PlotBandLine/PlotBand.integration.spec.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import Highcharts from 'highcharts';
+import {
+  HighchartsChart,
+  HighchartsProvider,
+  PlotBand,
+  YAxis,
+  XAxis,
+  useAxis,
+  LineSeries
+} from '../../../src';
+import { uuidRegex } from '../../test-utils';
+
+describe('<PlotBand /> integration', () => {
+  const AxisSpy = ({ axisId, axisRef }) => {
+    const axis = useAxis(axisId);
+    const addPlotBandOrLineSpy = jest.spyOn(axis, 'addPlotBandOrLine');
+    const removePlotBandOrLineSpy = jest.spyOn(axis, 'removePlotBandOrLine');
+
+    axisRef.current = axis;
+    axisRef.addPlotBandOrLineSpy = addPlotBandOrLineSpy;
+    axisRef.removePlotBandOrLineSpy = removePlotBandOrLineSpy;
+    return null;
+  };
```

Comment:
In the future this could be extracted into a separate component along with ChartSpy and SeriesSpy. 

Useful for writing integration tests.

Reply:
Probably worth doing now? Seems like something that will end up being forgotten otherwise